src/main/java/com/github/wovnio/wovnjava/Lang.java

```diff
@@ -1,94 +1,58 @@
+        map.put("nl",     new Lang("nl", "nl", "Dutch", "Nederlands"));
+        map.put("en",     new Lang("en", "en", "English", "English"));
+        map.put("fi",     new Lang("fi", "fi", "Finnish", "Suomi"));
+        map.put("fr",     new Lang("fr", "fr", "French", "Français"));
+        map.put("de",     new Lang("de", "de", "German", "Deutsch"));
+        map.put("el",     new Lang("el", "el", "Greek", "Ελληνικά"));
+        map.put("he",     new Lang("he", "he", "Hebrew", "עברית"));
+        map.put("id",     new Lang("id", "id", "Indonesian", "Bahasa Indonesia"));
+        map.put("it",     new Lang("it", "it", "Italian", "Italiano"));
+        map.put("ja",     new Lang("ja", "ja", "Japanese", "日本語"));
+        map.put("ko",     new Lang("ko", "ko", "Korean", "한국어"));
+        map.put("ms",     new Lang("ms", "ms", "Malay", "Bahasa Melayu"));
+        map.put("my",     new Lang("my", "my", "Burmese", "ဗမာစာ"));
+        map.put("ne",     new Lang("ne", "ne", "Nepali", "नेपाली भाषा"));
+        map.put("no",     new Lang("no", "no", "Norwegian", "Norsk"));
+        map.put("pl",     new Lang("pl", "pl", "Polish", "Polski"));
+        map.put("pt",     new Lang("pt", "pt", "Portuguese", "Português"));
+        map.put("ru",     new Lang("ru", "ru", "Russian", "Русский"));
+        map.put("es",     new Lang("es", "es", "Spanish", "Español"));
+        map.put("sv",     new Lang("sv", "sv", "Swedish", "Svensk"));
+        map.put("th",     new Lang("th", "th", "Thai", "ภาษาไทย"));
+        map.put("hi",     new Lang("hi", "hi", "Hindi", "हिन्दी"));
+        map.put("tr",     new Lang("tr", "tr", "Turkish", "Türkçe"));
+        map.put("uk",     new Lang("uk", "uk", "Ukrainian", "Українська"));
+        map.put("vi",     new Lang("vi", "vi", "Vietnamese", "Tiếng Việt"));
+        map.put("tl",     new Lang("tl", "tl", "Tagalog", "Tagalog"));
         LANG = Collections.unmodifiableMap(map);
     }
 
-    String name;
     String code;
-    String en;
-
-    Lang(String n, String c, String e) {
-        super();
-        this.name = n;
-        this.code = c;
-        this.en = e;
-    }
+    String hreflangCode;
 
-    @Contract("null -> null")
-    static String getCode(String langName) {
-        if (langName == null || langName.length() == 0) {
-            return null;
-        }
-        if (LANG.get(langName) != null) {
-            return langName;
-        }
-        for (Map.Entry<String, Lang> e : LANG.entrySet()) {
-            Lang l = e.getValue();
-            String langNameLC = langName.toLowerCase();
-            if ( langNameLC.equals(l.name.toLowerCase())
-                    || langNameLC.equals(l.en.toLowerCase())
-                    || langNameLC.equals(l.code.toLowerCase())
-                    ) {
-                return l.code;
-            }
-        }
-        return null;
+    Lang(String code, String hreflangCode, String englishName, String nativeName) {
```

Comment:
I would like not to loose the information that `hreflangCode` is actually the ISO639_1 code, so I would prefer you to rename `hreflangCode`, please.

Reply:
Adressed in https://github.com/WOVNio/wovnjava/pull/118

Reply:
thanks